lib/main.js

```diff
@@ -60,6 +60,7 @@ export default {
     const Composer = require('./composer')
 
     global.latex = new Latex()
+    latex.opener.initializeOpeners()
```

Comment:
It was done in `OpenerRegistry` but that caused problems in the spec since the schema was not available.